src/main/kotlin/no/nav/familie/ef/mottak/api/SøknadController.kt

```diff
@@ -46,6 +43,17 @@ class SøknadController(val søknadService: SøknadService) {
         return ResponseEntity.ok(søknadService.mottaBarnetilsyn(søknad, vedleggData))
     }
 
+    @PostMapping(consumes = [MULTIPART_FORM_DATA_VALUE], path = ["skolepenger"])
```

Comment:
Tror consumes = [MULTIPART_FORM_DATA_VALUE] kan flyttes til @RequestMapping for alle metodene.